src/types.ts

```diff
@@ -1,28 +1,32 @@
+export type Position = 'GK' | 'DF' | 'MF' | 'FW';
+
 export type PlayerInfo =
   | {
-      score: {
+      score?: {
         'round-1'?: number;
         'round-2'?: number;
       } | null;
-      club: string;
-      country: string;
-      name: string;
+      club?: string;
```

Comment:
Why make all of these optional – are they ever missing?

Reply:
Yeah, I kept making one by one optional until typescript stopped complaining! Idk how it's possible to have players without name, but apparently the data file stopped matching the un-optional type in a recent update

Reply:
Seems to me like it could perhaps happen if one of the two files is not properly pulled at all? I still haven't ran into this with any file I pulled 🤔

I think it would be nice to keep these as required of they are indeed that, to svoid having to write too much code to account for them being missing when they never are.

Reply:
I made them mandatory again, and now it seems like CI agrees with my editor that several of these can be undefined (but I can't find any that actually are missing scores or names in the actual data)